meta-phosphor/conf/machine/include/obmc-bsp-common.inc

```diff
@@ -13,7 +13,7 @@ UBOOT_SUFFIX ?= "bin"
 
 MACHINEOVERRIDES =. "openbmc:"
 
-IMAGE_FSTYPES += "overlay"
+MACHINE_FEATURES += "image-overlay"
```

Comment:
Hi! 

Have  you already read my  new article about the  elections? You've  got  to take a look, here is  the link <http://pychitrumu.thewitchinthewoods.net/e4kwts>

Yours faithfully, Brad Bishop



-- 
Esta mensagem foi verificada pelo sistema de antiv�rus e
 acredita-se estar livre de perigo.